google-cloud-storage/acceptance/storage/file_test.rb

```diff
@@ -47,7 +47,7 @@
   end
 
   after do
-    bucket.files.all { |f| f.delete rescue nil }
+    bucket.files(versions: true).all { |f| f.delete generation: true rescue nil }
```

Comment:
Is this needed to delete the file completely (including previous versions)?

The [DELETE Object](https://cloud.google.com/storage/docs/xml-api/delete-object) docs state:

> If this query parameter is used on a versioned bucket and the object is the current one, no archive of the object is made, but instead the object is deleted completely.